src/presenters/api/routers/pagamentoRouter.ts

```diff
@@ -1,48 +1,67 @@
 import express from "express";
 import { Request, Response } from "express";
 
-import MetodoPagamentoDatabaseRepository from "~datasources/database/repository/metodoPagamentoDatabaseRepository";
-import MetodoPagamentoUseCase from "~domain/useCases/metodoPagamentoUseCase";
+import PagamentoDatabaseRepository from "~datasources/database/repository/pagamentoDatabaseRepository";
+import { PagamentoController } from "~interfaceAdapters/controllers/pagamentoController";
 
-import { ListaPagamentosParams, ListaPagamentosPayload, ListaPagamentosSchema } from "./schemas/pagamentoRouter.schema";
+import { RecebimentoDePagamentosPayload, RecebimentoDePagamentosSchema } from "./schemas/pagamentoRouter.schema";
 import { validaRequisicao } from "./utils";
 
-const metodoPagamento = express.Router();
+const pagamentoRouter = express.Router();
+
+const dbPagamentoRepository = new PagamentoDatabaseRepository();
 
-const dbMetodoPagamentoRepository = new MetodoPagamentoDatabaseRepository();
 
 /**
  * @openapi
- * /metodoPagamento:
- *   get:
- *     summary: Lista metodos de pagamento
+ * /pagamento:
+ *   post:
+ *     summary: Recebe confirmação ou negação de pagamento
```

Comment:
```suggestion
 *     summary: WebHook - Recebe confirmação ou negação de pagamento
```

Apenas para deixar claro que é o webhook